### ISSUE TITLE
When using incline.nvim, an error always occurs when I close Neovim

### ISSUE THREAD

**Issue by tadashi-aikawa** [shortcuts/no-neck-pain.nvim#352]
## Description

Thank you for the great plugin!

After upgrading to version 1.12.2, I consistently encounter an error when exiting Neovim.

This problem is likely related to the following issue.

https://github.com/shortcuts/no-neck-pain.nvim/issues/348

## Steps to reproduce

1. Install https://github.com/b0o/incline.nvim
2. `nvim`
3. `:q`
4. Error

```
Error executing vim.schedule lua callback: ...re/nvim/lazy/no-neck-pain.nvim/lua/no-neck-pain/wins.lua:53: Vim:E444: Cannot close last window
stack traceback:
	[C]: in function 'nvim_win_close'
	...re/nvim/lazy/no-neck-pain.nvim/lua/no-neck-pain/wins.lua:53: in function 'close'
	...re/nvim/lazy/no-neck-pain.nvim/lua/no-neck-pain/main.lua:449: in function <...re/nvim/lazy/no-neck-pain.nvim/lua/no-neck-pain/main.lua:425>
```

## Expected behavior

Close Nvim without errors.

## Environment

- Neovim version: 0.9.5, 0.10.
- no-neck-pain.nvim version: 1.12.2 (No error occurs in version 1.12.1.)
- incline.nvim version: `16fc9c0`

**Comment by shortcuts** [comment]
Hey there, thanks a lot for using the plugin and reporting the issue! I'm working on a fix right now, sorry for the issue

**Comment by shortcuts** [comment]
Hey, this should now be fixed in 1.22.3, feel free to let me know how it went!

**Comment by tadashi-aikawa** [comment]
@shortcuts 
Thank you for your prompt response! I have confirmed that the issue has been fixed👍
